packages/experiments/src/components/Chiclet/Chiclet.types.ts

```diff
@@ -0,0 +1,180 @@
+import * as React from 'react';
+import { BaseChiclet } from './BaseChiclet';
+import { ChicletCard } from './ChicletCard';
+import { Chiclet } from './Chiclet';
+import { IStyle, ITheme } from '../../Styling';
+import { IButtonProps } from 'office-ui-fabric-react/lib/Button';
+import { IStyleFunction } from '../../Utilities';
+
+export interface IChiclet {
+
+}
+
+export interface IChicletStyles {
+  /**
+   * Style for the root element.
+   */
+  root?: IStyle;
+
+  /**
+   * Style for the icon that overlays the file preview image.
+   */
+  icon?: IStyle;
+
+  /**
+   * Style for the file preview image.
+   */
+  preview?: IStyle;
+
+  /**
+   * Style for preview information about the file, such as title and link.
+   */
+  info?: IStyle;
+
+  /**
+   * Style for the title of the file.
+   */
+  title?: IStyle;
+
+  /**
+   * Style for the link to the file.
+   */
+  link?: IStyle;
+
+  /**
+   * Style for the collective actions in the lower right of the component.
+   */
+  actions?: IStyle;
+
+  /**
+   * Style for the individual actions in the lower right of the component.
+   */
+  action?: IStyle;
+}
+
+export interface IChicletStyleProps {
+
+}
+
+export interface IBaseChicletProps extends React.Props<BaseChiclet> {
+  /**
+   * Optional callback to access the IChiclet interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IChiclet | null) => void;
+
+  /**
+   * Optional class for chiclet.
+   */
+  className?: string;
+
+  /**
+   * Sharing link
+   */
+  url: string;
+
+  /**
+   * Chiclet size to render
+   */
+  size?: string;
```

Comment:
Can we leverage the enum `ChicletType` here for type-safety? I know TS is a bit odd with key/value for enums, but it should be do-able.